tests/schema_registry/test_proto.py

```diff
@@ -83,3 +111,27 @@ def test_index_encoder(msg_idx, zigzag, expected_hex):
     buf.seek(0)
     decoded_msg_idx = ProtobufDeserializer._decode_index(buf, zigzag=zigzag)
     assert decoded_msg_idx == msg_idx
+
+
+@pytest.mark.parametrize("pb2, data", TEST_DATA)
+def test_round_trip(mock_schema_registry, pb2, data):
+
+    conf = {"use.deprecated.format": False}
+    test_client = mock_schema_registry({"url": TEST_URL})
+    context = SerializationContext("topic-name", MessageField.VALUE)
+    serializer = ProtobufSerializer(pb2, test_client, conf=conf)
+    deserializer = ProtobufDeserializer(pb2, conf=conf)
+
+    message_before = pb2(**data)
+    bytes_before = serializer(message_before, context)
+    message_after = deserializer(bytes_before, context)
+    bytes_after = serializer(message_after, context)
+    assert isinstance(message_after, pb2)
+
+    assert message_before == message_after
+    assert bytes_after == bytes_before
+
+
+def test_foo_bar():
```

Comment:
We are removing this code altogether, so I think we don't need to test this. Please remove this function.